test/application_system_test_case.rb

```diff
@@ -11,7 +11,7 @@ class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
   driven_by :cuprite, using: :chrome, screen_size: [1400, 1400]
 
   def with_preview(preview_name)
-    component_uri = self.class.name.split("Integration").first.underscore
+    component_uri = self.class.name.gsub("Test", "").gsub("Integration", "").underscore
```

Comment:
Could you help me understand this change?

Reply:
I moved the integration tests from `test/components/my_component_integration_test.rb` to `test/integration/my_component_test.rb`
so the class name was `MyComponentIntegrationTest` and now is `IntegrationMyComponentTest`, which changes how we extract the component name